ntbs-service/Pages/Notifications/Edit/HospitalDetails.cshtml.cs

```diff
@@ -194,7 +205,7 @@ public async Task<JsonResult> OnGetGetFilteredListsByTbService(string value)
                 await GetActiveOrCurrentHospitalsByTbServiceCodesAsync(tbServiceCodeAsList, notification);
 
             var filteredCaseManagers =
-                await _referenceDataRepository.GetCaseManagersByTbServiceCodesAsync(tbServiceCodeAsList);
+                await _referenceDataRepository.GetActiveCaseManagersByTbServiceCodesAsync(tbServiceCodeAsList);
```

Comment:
I'm assuming this is called when a new value is selected in the TB Service? So this means that if you select a different TB Service, and then switch back to the original one, you won't be able to select the original case manager?

I think this is probably ok... what do you think?

Reply:
There's an argument for both sides of this. I guess if you accidentally changed the TB service and then realised you made a mistake you would just leave the page without saving, which would leave the case manager as is. 
Making the change to include the current case manager here would just mean changing this line to use the same method that's used in populate dropdowns so it's just whether we think we want that option.

Reply:
I don't think it would be that simple actually, because the other method always adds the existing case manager to the list, whereas we'd only want to do it for one TB service. To avoid over-complicating this, I think we should stick with what we've got. (As you suggest, if this happens the user will have to leave without saving and come back).